### PR TITLE
Sarkars/update benchmarks

### PR DESCRIPTION
**Pull request by sayantan-nervana**

### CODE REVIEW

**Comment by sayantan-nervana** [comment]
We should also consider the mnist example and make it fully trainable and inferenceable out-of-the-box. for that we might need to ask the user to download data, something like https://pypi.org/project/python-mnist/

File: test/ci/run-premerge-ci-checks.sh [hunk 1]
```diff
@@ -68,14 +68,23 @@ echo "--------------------------------------------------------------------------
...
-  --data_format NCHW \
-  --data_name=imagenet --data_dir "${NGRAPH_IMAGENET_DATASET}" --datasets_use_prefetch=False 
+#${JUNIT} python tf_cnn_benchmarks.py --model=resnet50 --eval --num_inter_threads=1 \
+#  --batch_size=16 --num_batches=50 \
+#  --train_dir "${NGRAPH_TRAINED_MODEL}" \
+#  --data_format NCHW \
+#  --data_name=imagenet --data_dir "${NGRAPH_IMAGENET_DATASET}" --datasets_use_prefetch=False 
+
+# Training test
+${JUNIT} python tf_cnn_benchmarks.py --batch_size=128 --model=resnet50 --num_inter_threads=1 --train_dir=./modelsavepath/ --num_batches 10
```
[the remark this answers is not in the document]
Comment (sayantan-nervana) [reply test/ci/run-premerge-ci-checks.sh]: Can we put it in a separate PR (using Mingshan's framework)

[the remark this answers is not in the document]
Comment (avijit-nervana) [reply test/ci/run-premerge-ci-checks.sh]: Yes - definitely.

**Comment by sayantan-nervana** [comment]
@avijit-nervana : This is ready.

**Pull request merged**